tests/unit/plugins/module_utils/cloud/test_scaleway.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) Ansible project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import random
+
+from ansible_collections.community.general.tests.unit.compat import unittest
+from ansible_collections.community.general.plugins.module_utils.scaleway import SecretVariables
```

Comment:
```suggestion
from ansible_collections.community.general.plugins.module_utils.scaleway import SecretVariables, argon2
```
and skip the tests that require argon2 if `argon2 is None`.

Reply:
The issue should be fixed by 9dd8ae7a0314b2b5eff226768b3d766a3cd14544